lib/utils/telemetry/generatePayload.js

```diff
@@ -298,6 +299,17 @@ module.exports = ({
     payload.config = getServiceConfig({ configuration, options, variableSources });
     payload.isConfigValid = getConfigurationValidationResult(configuration);
     payload.dashboard.orgUid = serverless && serverless.service.orgUid;
+
+    if (command === 'deploy' && isAwsProvider) {
+      const serviceName = configuration && configuration.service.name;
+      const accountId = serverless && serverless.getProvider('aws').accountId;
```

Comment:
I would probably put `&& serverless` in main condition, and assume it unconditionally here.

Reply:
changed